citrixitm/resource_citrixitm_dns_app_test.go

```diff
@@ -208,49 +222,49 @@ func testAccCheckCitrixITMDnsAppConfig(randString string) string {
 resource "citrixitm_dns_app" "foo" {
   name 				= "foo-%s"
   description		= "some description"
-  app_data			= "// some source"
+  app_data			= "%s"
   fallback_cname	= "fallback.foo.com"
-}`, randString)
+}`, randString, escapedMinimalAppSource)
 }
 
 func testAccCheckCitrixITMDnsAppConfig_Rename(randString string) string {
 	return fmt.Sprintf(`
 resource "citrixitm_dns_app" "foo" {
   name 				= "bar-%s"
   description		= "some description"
-  app_data			= "// some source"
+  app_data			= "%s"
   fallback_cname	= "fallback.foo.com"
-}`, randString)
+}`, randString, escapedMinimalAppSource)
 }
 
 func testAccCheckCitrixITMDnsAppConfig_ChangeAppData(randString string) string {
 	return fmt.Sprintf(`
 resource "citrixitm_dns_app" "foo" {
   name 				= "bar-%s"
   description		= "some description"
-  app_data			= "// some source foo"
+  app_data			= "%s"
```

Comment:
Formatting seems odd here, but seems like it was odd before too.

Ah, just checked it in my editor and when I set tab width to 4, it looks fine.